Add missing README entry for es:migrate command (#89)
kuzzleio/kourou: README.md

_See code: [src/commands/es/indices/insert.ts](src/commands/es/indices/insert.ts)_

## `kourou es:migrate`

> Migrate all the index from one Elasticsearch server to another

Reviewer: ```suggestion
Migrate all the indexes from one Elasticsearch server to another
```

Author: I'll update it in the migrate.ts file and generate a new README

```
USAGE
  $ kourou es:migrate

OPTIONS

>   --batch-size=batch-size  [default: 1000] How many documents to move in batch per operation

Reviewer: ```suggestion
  --batch-size=batch-size  [default: 1000] The amount of documents to move in batch per operation
```

Author: I'll update it in the migrate.ts file and generate a new README
